applications/StructuralMechanicsApplication/tests/cpp_advanced_constitutive_tests/test_rule_of_mixtures_law.cpp

```diff
@@ -55,7 +55,7 @@ Parameters GetTwoLayersParameters()
         "properties_id"   : 1,
         "Material"        : {
             "constitutive_law" : {
-                "name" : "ParallelRuleOfMixturesLaw",
+                "name" : "ParallelRuleOfMixturesLaw3D",
```

Comment:
I've changed the name here! should be fine

Reply:
Check three layers:
![imagen](https://user-images.githubusercontent.com/19991680/83252996-82418400-a1ac-11ea-8ff2-61b6ad4e1eaa.png)